extra/cockpit/foreman-cockpit.service

```diff
@@ -1,14 +1,14 @@
 [Unit]
 Description=Foreman Cockpit
-Documentation=https://theforeman.org
+Documentation=Websocket authentication service for Foreman-Cockpit integration
```

Comment:
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Documentation= states this is invalid.
> A space-separated list of URIs referencing documentation for this unit or its configuration. Accepted are only URIs of the types "http://", "https://", "file:", "info:", "man:".

Reply:
I accidentally swapped this with Description actually. Fixing.